packages/ketcher-react/src/script/editor/tool/sgroup.ts

```diff
@@ -473,6 +458,11 @@ class SGroupTool {
     const type = sg ? sg.type : defaultType
     const eventName = type === 'DAT' ? 'sdataEdit' : 'sgroupEdit'
 
+    if (!selection.atoms && !selection.bonds && !sg) {
+      console.info('There is no selection or sgroup')
```

Comment:
Do we really need this `console.info`?

Reply:
[Delete console.info](https://github.com/epam/ketcher/pull/2007/commits/7ba5fe6959c05a1c7ab4cdf0f0e92e92ceddae86)
[7ba5fe6](https://github.com/epam/ketcher/pull/2007/commits/7ba5fe6959c05a1c7ab4cdf0f0e92e92ceddae86)